eZ/Publish/Core/Persistence/Legacy/Content/Type/Gateway/DoctrineDatabase.php

```diff
@@ -130,7 +130,7 @@ public function __construct(DatabaseHandler $db, Connection $connection, MaskGen
      *
      * @param \eZ\Publish\SPI\Persistence\Content\Type\Group $group
      *
-     * @return mixed Group ID
+     * @return int Group ID
      */
     public function insertGroup(Group $group)
```

Comment:
Anything stopping us now from adding return type?
```suggestion
    public function insertGroup(Group $group): int
```